UefiTestingPkg/AuditTests/BootAuditTest/UEFI/BootAuditTestApp.c

```diff
@@ -31,28 +31,141 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include <Library/UefiLib.h>
 #include <Library/UefiRuntimeServicesTableLib.h>
 #include <Library/UnitTestLib.h>
+#include <Library/DevicePathLib.h>
+#include <Library/UefiBootManagerLib.h>
+#include <Library/UefiBootServicesTableLib.h>
+#include <Library/FileHandleLib.h>
 
-#define UNIT_TEST_APP_NAME     "BootAuditTestApp"
-#define UNIT_TEST_APP_VERSION  "1.0"
+#define UNIT_TEST_APP_NAME      "BootAuditTestApp"
+#define UNIT_TEST_APP_FILENAME  L"BootAuditTestApp.efi"
+#define UNIT_TEST_APP_VERSION   "1.0"

```

Comment:
I suggest bumping the version. Easier to say "v1.1 has the change" later.

Reply:
Bumped to v1.1